core/src/main/java/hudson/model/AdministrativeMonitor.java

```diff
@@ -118,8 +118,12 @@ public final String getSearchUrl() {
     public void disable(boolean value) throws IOException {
         AbstractCIBase jenkins = Jenkins.get();
         Set<String> set = jenkins.getDisabledAdministrativeMonitors();
-        if(value)   set.add(id);
-        else        set.remove(id);
+        if (value) {
+            set.add(id);
+        } else {
+            set.remove(id);
+        }
```

Comment:
This seems dangerous, you should check if minor is there before trying to add or remove it.

Reply:
Never mind, hashset is safe to remove or add:
http://hg.openjdk.java.net/jdk8/jdk8/jdk/file/687fd7c7986d/src/share/classes/java/util/HashSet.java